src/Platform/Microsoft.Testing.Extensions.OpenTelemetry/Microsoft.Testing.Extensions.OpenTelemetry.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;$(MicrosoftTestingTargetFrameworks)</TargetFrameworks>
```

Comment:
MicrosoftTestingTargetFrameworks property was removed in https://github.com/microsoft/testfx/pull/6657.

This should switch to `SupportedNetFrameworks`